src/runner/mos6502.rs

```diff
@@ -69,3 +70,20 @@ pub unsafe fn run(output: *mut String_Builder, config: Config, output_path: *con
 
     Some(())
 }
+
+pub unsafe fn run_async(config: Config, output_path: *const c_char) -> Proc {
+    let pid = fork();
```

Comment:
Ok, how to do this on Windows?

Reply:
Does the b compiler support the Windows platform? As I understood from the code that works with Linux utilities, like qemu, etc. - the b compiler only works on Unix systems, and already compiled binary files for window can be run either in wine or on the Windows operating system.

Or is it not so?

Reply:
Oh... I try to compile btest.exe and I got the `undefined reference to `fork'` error. 
Give me some time to fix this.

Reply:
I was able to make it so that `fork` is not used and `btest` is built for windows. However, if I run tests in parallel, I get instability when running tests in wine. Periodically one or another test for window platform terminates with an error. I think it is necessary to reconsider the approach to parallel testing.

I am closing this PR. Thanks!